### PR TITLE
Move messages above search to new status area in search controls header

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | -
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Selection_244
Selection_243

### CODE REVIEW

File: templates/components/search/controls.html.twig [hunk 1]
```diff
@@ -95,12 +95,15 @@
...
 
-
         {% if mainform and showaction %}
-        <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
-        </form>
+            <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
+            </form>
         {% endif %}
 
+        <div class="status-area">
```
Comment (orthagh) [inline templates/components/search/controls.html.twig:103]: Maybe something like that to draw attention:

```suggestion
        <div class="status-area alert alert-danger p-1 m-0">
```

image

**Review by orthagh — APPROVED** [review]

File: src/CronTask.php [hunk 2]
```diff
@@ -2004,4 +2004,46 @@ public static function getIcon()
...
+            $CFG_GLPI['cron_limit'] < countElementsInTable(
+                'glpi_crontasks',
+                ['frequency' => MINUTE_TIMESTAMP]
+            )
+        ) {
+            $warnings[] = __("You have more actions scheduled to run every minute than the number allowed to run at once. Increase this config.");
+        }
+
+        if (count($warnings) > 0) {
+            $msg = __s('Automatic actions may not be running as expected');
```
Comment (cedric-anne) [inline src/CronTask.php:2033]: Will be escaped by Twig engine.
```suggestion
            $msg = __('Automatic actions may not be running as expected');
```

File: src/CronTask.php [hunk 3]
```diff
@@ -2004,4 +2004,46 @@ public static function getIcon()
...
+            $params = [
+                'msg' => $msg,
+                'warnings' => '<ul>' . implode('', array_map(static fn ($warning) => "<li>$warning</li>", $warnings)) . '</ul>'
```
Comment (cedric-anne) [inline src/CronTask.php:2036]:
```suggestion
                'warnings' => '<ul>' . implode('', array_map(static fn ($warning) => '<li>' . htmlspecialchars($warning) . '</li>', $warnings)) . '</ul>'
```

File: src/MailCollector.php [hunk 4]
```diff
@@ -1869,11 +1880,20 @@ public function title()
...
         }
-        TemplateRenderer::getInstance()->display('pages/setup/mailcollector/results_title.html.twig', [
-            'servers' => $servers,
-            'errors' => $errors,
-            'has_collector' => countElementsInTable(self::getTable()) > 0
-        ]);
+
+        if (count($servers)) {
+            $server_links = implode(' ', array_map(
+                static fn ($v) => '<a class="btn btn-sm btn-ghost-danger align-baseline" href="' . $v['link'] . '">' . htmlentities($v['name']) . '</a>',
```
Comment (cedric-anne) [inline src/MailCollector.php:1886]:
```suggestion
                static fn ($v) => '<a class="btn btn-sm btn-ghost-danger align-baseline" href="' . htmlspecialchars($v['link']) . '">' . htmlspecialchars($v['name']) . '</a>',
```

File: src/MailCollector.php [hunk 5]
```diff
@@ -1854,10 +1862,13 @@ public function sendMailRefusedResponse($to = '', $subject = '')
         $mmail->send();
     }
 
-
-    public function title()
```
Comment (cedric-anne) [inline src/MailCollector.php:1858]: Please add an entry in changelog to indicate that this method was removed.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Pull request merged**